extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php

```diff
@@ -25,6 +25,12 @@ public function appreciate() {
 			// send email that user received appreciation.
 			// Currently disabled because of: https://wikia-inc.atlassian.net/browse/WW-172
 			// $this->sendMail( $revisionId );
+
+			\Wikia\Logger\WikiaLogger::instance()->info('ContributionAppreciationMessage email send', [
+				'revision_id' => $revisionId,
+				'wiki_id' => $wgCityId,
+				'user_id' => $wgUser->getId()
```

Comment:
This is logged by default, see:
Wikia\Logger\Hooks::onWikiFactoryExecute
and
Wikia\Tracer\WikiaTracer::updateContext